test/parallel/test-net-listen-anyaddr.js

```diff
@@ -0,0 +1,10 @@
+var assert = require('assert');
+var cp = require('child_process');
+
+var args = [
+  '-e',
+  "require('net').createServer().listen(0, function() { this.close(); });",
+];
+
+// should exit cleanly almost immediately
+cp.execFile(process.execPath, args, {timeout: 500}, assert.ifError);
```

Comment:
500 is definitely too low for ARMv6, where a simple startup can take upwards of 3 seconds. I suggest using `common.platformTimeout(1000)`, which right now applies factor 7 (7 seconds) for ARMv6.